core/lib/spree/migration.rb

```diff
@@ -0,0 +1,4 @@
+module Spree
+  class Migration < ActiveRecord::Migration[ActiveRecord::Migration.current_version]
```

Comment:
I'm worried leaving this dynamic is going to cause some headaches in the future. While true that we don't need to worry about existing stores, since they've already copied in the migrations, any future changes to `ActiveRecord::Migration` will result in non-deterministic migrations for stores. 

And possibly even more overhead for existing users. If we find ourselves needing to update old migrations to work with more recent versions of Rails, any stores that have already copied those migrations will likely need to also copy over our changes themselves. (Or just never run migrations again, but I don't think that's something we can unilaterally enforce.)

I think @BenMorganIO's approach is the more sustainable one. We should provide migrations that can help update existing tables to match the new defaults we'd get from running the migration on more modern versions of Rails. That will help ensure both existing and new Solidus stores have a consistent schema to work from.